frontend/src/App.tsx

```diff
@@ -94,6 +113,15 @@ function Mayournaise() {
           </p>
         </div>
 
+        {/* Added Randomize Button */}
+        <button
+          type="button" // Important: Set type to "button" to prevent form submission
+          onClick={handleRandomize}
+          className="w-full py-3 px-4 font-semibold rounded-lg shadow-md focus:outline-none focus:ring-2 focus:ring-green-400 focus:ring-opacity-75 text-sm sm:text-base mt-2 bg-green-500 hover:bg-green-600 text-white"
+        >
+          Randomize Options
```

Comment:
Can you change this to just "Randomize!" with maybe an emoji of a magic wand please